plugins/removeHiddenElems.js

```diff
@@ -350,46 +393,44 @@ exports.fn = (root, params) => {
           removeElement(node, parentNode);
         }
       },
-
-      exit: (node, parentNode) => {
-        if (node.name === 'defs' && node.children.length === 0) {
-          removeElement(node, parentNode);
-          return;
-        }
-
-        if (node.name === 'use') {
-          const referencesRemovedDef = Object.entries(node.attributes).some(
-            ([attrKey, attrValue]) =>
-              (attrKey === 'href' || attrKey.endsWith(':href')) &&
-              removedDefIds.has(
-                attrValue.slice(attrValue.indexOf('#') + 1).trim()
-              )
-          );
-
-          if (referencesRemovedDef) {
-            detachNodeFromParent(node, parentNode);
+    },
+    root: {
+      exit: () => {
+        // Remove uses of deleted definitions
```

Comment:
```suggestion
```

Sorry for not being explicit, could we remove these comments too?
The code is all very clear already.